projects/angular/src/data/stack-view/_stack-view.clarity.scss

```diff
@@ -199,154 +70,77 @@
-        background-color,
-        clr-stack-view-stack-block-label-and-content-bg-color,
-        stack-view-variables.$clr-stack-view-stack-block-label-and-content-bg-color,
-        variables.$clr-use-custom-properties
-      );
-    }
-
-    .stack-block-content {
-      background-color: inherit;
+    .stack-block-changed::before {
+      content: ' ';
+      position: absolute;
+      @include mixins.equilateral(0);
+      border-top: tokens.$cds-global-space-5 solid stack-view-variables.$clr-stack-block-changed-border-top-color;
+      border-right: tokens.$cds-global-space-5 solid transparent;
     }
 
     .stack-block-caret {
       align-self: center;
     }
 
     .stack-block-label {
-      @include mixins.css-var(
-        color,
-        clr-stack-view-stack-block-label-text-color,
-        stack-view-variables.$clr-stack-view-stack-block-label-text-color,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-size,
-        clr-stack-view-stack-block-label-font-size,
-        stack-view-variables.$clr-stack-view-stack-block-label-font-size,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-weight,
-        clr-stack-view-stack-block-label-font-weight,
-        stack-view-variables.$clr-stack-view-stack-block-label-font-weight,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        line-height,
-        clr-stack-view-stack-block-label-line-height,
-        stack-view-variables.$clr-stack-view-stack-block-label-line-height,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        letter-spacing,
-        clr-stack-view-stack-block-label-letter-spacing,
-        stack-view-variables.$clr-stack-view-stack-block-label-letter-spacing,
-        variables.$clr-use-custom-properties
+      padding: tokens.$cds-global-space-5 tokens.$cds-global-space-6;
+      color: stack-view-variables.$clr-stack-view-row-color;
+      background-color: stack-view-variables.$clr-stack-view-row-bg-color;
+
+      @include mixins.generate-typography-token(
+        'SECONDARY-13-RG-STD',
+        (
+          font-size: stack-view-variables.$clr-stack-view-row-font-size,
+          font-weight: stack-view-variables.$clr-stack-view-row-font-weight,
+          line-height: stack-view-variables.$clr-stack-view-row-line-height,
+          letter-spacing: stack-view-variables.$clr-stack-view-row-letter-spacing,
+        )
       );
 
-      // We're doing a non-flexible item.
-      // It's just easier to align and stretch everything vertically this way than with
-      // floats or inline-blocks.
       flex: 1 1 auto;
       display: flex;
+      align-items: center;
 
       // ATM
       // before section is caret placeholder remove it and replace it with pure padding right
-      // padding-right: var(--cds-global-space-9);
+      // padding-right: tokens.$cds-global-space-9;
```

Comment:
Just remove maybe?

Reply:
I kept this ATM comment for refactoring JIRA investigation, but can remove it altogether.

Reply:
comment removed in commit be6430d